runtime/acala/src/lib.rs

```diff
@@ -1755,24 +1755,53 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	XcmInterfaceMigration,
+	TransactionPaymentMigration,
 >;
 
-pub struct XcmInterfaceMigration;
-impl OnRuntimeUpgrade for XcmInterfaceMigration {
+parameter_types! {
+	pub FeeTokens: Vec<CurrencyId> = vec![AUSD, DOT, LDOT, LCDOT];
+}
+pub struct TransactionPaymentMigration;
+impl OnRuntimeUpgrade for TransactionPaymentMigration {
 	fn on_runtime_upgrade() -> frame_support::weights::Weight {
-		let _ = <module_xcm_interface::Pallet<Runtime>>::update_xcm_dest_weight_and_fee(
-			Origin::root(),
-			vec![(
-				module_xcm_interface::XcmInterfaceOperation::ParachainFee(Box::new(
-					(1, Parachain(parachains::statemint::ID)).into(),
-				)),
-				Some(4_000_000_000),
-				Some(50_000_000),
-			)],
-		);
+		let poo_size = 5 * dollar(ACA);
+		let threshold = Ratio::saturating_from_rational(1, 2).saturating_mul_int(dollar(ACA));
+		for token in FeeTokens::get() {
+			let _ = module_transaction_payment::Pallet::<Runtime>::disable_pool(token);
```

Comment:
why do we need to reinitialize the pool?

Reply:
`initialize_pool` method now don't store `GlobalFeeSwapPath` anymore, reinitialize pool here is prepare for migration. after migration we can remove `GlobalFeeSwapPath`.